src/components/WorkshopPage/Workshop.js

```diff
@@ -3,42 +3,88 @@ import ReactPlayer from 'react-player/youtube';
 import Button from '@material-ui/core/Button';
 import Grid from '@material-ui/core/Grid';
 import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
 import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import YouTubeIcon from '@material-ui/icons/YouTube';
+import GitHubIcon from '@material-ui/icons/GitHub';
+import SlideshowIcon from '@material-ui/icons/Slideshow';
 
-function Workshop(props) {
+const useStyles = makeStyles(theme => ({
+	button: {
+		textTransform: 'none',
+		padding: '4px 1.5em',
```

Comment:
Just to make things a little more consistent, in the `description` class, you have padding specified separately for each direction, while here you do it in a single property. For better readability, I would suggest changing this so that it's also specified separately. 

https://www.w3schools.com/css/css_padding.asp